extensions/wikia/AdEngine/js/lookup/prebid/adapters/appnexusWebAds.js

```diff
@@ -15,15 +15,67 @@ define('ext.wikia.adEngine.lookup.prebid.adapters.appnexusWebAds',[
 		priorityLevel = 0,
 		slots = {
 			oasis: {
+				/* WIKIA_LB_AS */
+				TOP_LEADERBOARD: {
+					placementId: '13104394',
+					sizes: [
+						[728, 90],
+						[970, 250]
+					]
+				},
+				/* WIKIA_MR_AS */
+				TOP_RIGHT_BOXAD: {
+					placementId: '13104393',
+					sizes: [
+						[300, 250],
+						[300, 600]
+					]
+				},
+				/* WIKIA_BLB_AS */
+				BOTTOM_LEADERBOARD: {
+					placementId: '13104395',
+					sizes: [
+						[728, 90],
+						[970, 250]
+					]
+				},
+				/* WIKIA_FMR_AS */
 				INCONTENT_BOXAD_1: {
 					placementId: '12939349',
 					sizes: [
-						[300, 600],
```

Comment:
I know it wasn't listed in file, but is it ok that we remove those sizes?

Reply:
Leaving removed.